src/controllers/update/confirm.to.remove.controller.ts

```diff
@@ -0,0 +1,71 @@
+import { NextFunction, Request, Response } from "express";
+
+import { Session } from "@companieshouse/node-session-handler";
+
+import { ApplicationData } from "../../model";
+import { findBeneficialOwner, getApplicationData } from "../../utils/application.data";
+
+import { logger } from "../../utils/logger";
+import {
+  CONFIRM_TO_REMOVE_PAGE,
+  PARAM_BENEFICIAL_OWNER_GOV,
+  PARAM_BENEFICIAL_OWNER_INDIVIDUAL,
+  PARAM_BENEFICIAL_OWNER_OTHER,
+  PARAM_BENEFICIAL_OWNER_TYPE,
+  UPDATE_BENEFICIAL_OWNER_TYPE_PAGE,
+  UPDATE_BENEFICIAL_OWNER_TYPE_URL
+} from "../../config";
+import { DoYouWantToRemoveKey } from "../../model/data.types.model";
+import { removeBeneficialOwnerIndividual } from "../../utils/beneficial.owner.individual";
+import { removeBeneficialOwnerGov } from "../../utils/beneficial.owner.gov";
+import { removeBeneficialOwnerOther } from "../../utils/beneficial.owner.other";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `${req.method} ${req.route.path}`);
+
+    const session = req.session as Session;
+    const appData: ApplicationData = getApplicationData(session);
+
+    return res.render(CONFIRM_TO_REMOVE_PAGE, {
+      backLinkUrl: UPDATE_BENEFICIAL_OWNER_TYPE_PAGE,
+      templateName: CONFIRM_TO_REMOVE_PAGE,
+      beneficialOwnerName: getBoName(appData, req.params[PARAM_BENEFICIAL_OWNER_TYPE], req.params['id'])
+    });
+  } catch (error) {
+    logger.errorRequest(req, error);
+    next(error);
+  }
+};
+
+export const post = (req: Request, res: Response, next: NextFunction) => {
```

Comment:
there's a remove endpoint(s) that were ported over by K2 some time ago that we're not using - we should delete those as a cleanup. 

We should call this endpoint remove/delete though, and in the logger we're going to get POST as the req.method but we want REMOVE/DELETE

Reply:
updated log message, will look into deleting the remove endpoint in each Update BO controller